nilearn/datasets/tests/test_func.py

```diff
@@ -372,7 +375,7 @@ def test_fetch_abide_pcp(tmp_path, request_mocker, quality_checked):
     )
 
 
-def test__load_mixed_gambles(request_mocker):
```

Comment:
How do you get rid of request_mocker ?

Reply:
it is set to autouse so it gets called in any case before any test

https://github.com/nilearn/nilearn/blob/0b49cc0cd99ca5bfd8780e5429c5d744dc21db81/nilearn/datasets/_testing.py#L69